asm/macros/battle_script.inc

```diff
@@ -2277,6 +2277,10 @@
 	various \battler, VARIOUS_ACTIVATE_TERRAIN_CHANGE_ABILITIES
 	.endm
 
+	.macro applyterastallization battler:req
+	various \battler, VARIOUS_APPLY_TERASTALLIZATION
+	.endm
+
```

Comment:
Can you move this to a callnative? We're slowly deprecating the use of `various`. An example would be `BS_HandleMegaEvolution` for example.

Reply:
Done in my latest commit 🙂